MediaBrowser.Model/GlobalSuppressions.cs

```diff
@@ -0,0 +1,8 @@
+// This file is used by Code Analysis to maintain SuppressMessage
+// attributes that are applied to this project.
+// Project-level suppressions either have no target or are given
+// a specific target and scoped to a namespace, type, member, etc.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("Performance", "CA1819:Properties should not return arrays", Justification = "Breaking Change")]
```

Comment:
I believe we have historically added the ignore to the csproj, but I like this method as it provides context